src/solver/conv_hip_implicit_gemm_bwd_v1r1_xdlops_nchw_kcyx_nkhw.cpp

```diff
@@ -753,6 +753,11 @@ ConvHipImplicitGemmBwdDataV1R1Xdlops::GetWorkspaceSize(const ConvolutionContext&
 
 bool ConvHipImplicitGemmBwdDataV1R1Xdlops::IsApplicable(const ConvolutionContext& ctx) const
 {
+#if WORKAROUND_SWDEV_251757
+    (void)ctx;
+    if(miopen::HipCompilerVersion() >= external_tool_version_t{3, 7, 0})
+        return false;
+#else
```

Comment:
[Notice] BTW now you can replace `#else` with `#endif`, and remove `(void)ctx`  and `#endif` at line 788.

Reply:
...just like shown at https://github.com/ROCmSoftwarePlatform/MIOpen/pull/457#pullrequestreview-493915098